trunk/research/hls/ts_info.cc

```diff
@@ -94,16 +94,16 @@ enum TSPidTable
 /**
 * Table 2-5 – Adaptation field control values, page 38.
 */
-enum TSAdaptionType
+enum TSAdaptationType
 {
 	// Reserved for future use by ISO/IEC
-	TSAdaptionTypeReserved		= 0x00,
+	TSAdaptationTypeReserved		= 0x00,
```

Comment:
I can fix whitespace if requested. It took at lot for me to convince myself that this was the correct fix but line 95 clearly indicates it is.